x/ecocredit/params.go

```diff
@@ -37,12 +41,38 @@ func validateCreditClassFee(i interface{}) error {
 	return nil
 }
 
-func NewParams(creditClassFee sdk.Coins) Params {
+func validateAllowlistCreditCreators(i interface{}) error {
```

Comment:
Can we add some tests for these?

Reply:
yup - [added some tests](https://github.com/regen-network/regen-ledger/pull/426/commits/4625bd5b5422a6bd12c3aa93b4b3255e7e3c5fb4) for the params and extended the test suite with some allowlist cases